test/local-ci-test.sh

```
@@ -20,8 +33,25 @@ npm link ../
 
 # Kill any existing process on port 8080 to prevent EADDRINUSE
 echo "Attempting to free port 8080..."
-fuser -k 8080/tcp || true
-
+#Try fuser(linux)
+if command -v fuser &>/dev/null; then
+    fuser -k 8080/tcp || true
+#Try lsof(macOs,some linux)
+elif command -v lsof &>/dev/null; then
+    PID=$(lsof -ti :8080)
+    if [-n "$PID "]; then
```

Shellcheck (0.10.0)

[error] You need a space after the [ and before the ].

(SC1035)